onyo/commands/shell_completion.py

```diff
@@ -478,55 +28,13 @@ def shell_completion(args: argparse.Namespace) -> None:
         $ source <(onyo shell-completion)
         $ onyo --<PRESS TAB to display available options>
     """
-    parser = args.parser
+    content = ''
+    shell_completion_dir = Path(Path(__file__).resolve().parent.parent, 'shell_completion')
```

Comment:
This would be more consistent with the rest of the code base (and I think easier to read).
Also: Slightly faster than invoking the `Path` constructor again IIRC.

```suggestion
    shell_completion_dir = Path(__file__).resolve().parent.parent / 'shell_completion'
```

Reply:
Thanks for the suggestion! I will update and push.